gradle.properties

```diff
@@ -1,14 +1,26 @@
-org.gradle.jvmargs=-Xmx4g
+org.gradle.caching=true
```

Comment:
I think most of these are enabled by default.

Reply:
Not sure this properity is enabled by default, bucause not found default value in gradle source